README.rst

```diff
@@ -17,17 +17,19 @@
 Scanpy – Single-Cell Analysis in Python
 =======================================
 
-Scanpy is a scalable toolkit for analyzing single-cell gene expression data
-built jointly with `anndata <https://anndata.readthedocs.io>`__. It includes
-preprocessing, visualization, clustering, trajectory inference and differential
-expression testing. The Python-based implementation efficiently deals with
-datasets of more than one million cells.
+Scanpy is a scalable toolkit for analyzing single-cell
+gene expression data built jointly with anndata_.
```

Comment:
This gave an error for me for some reason...

Reply:
Yup, exactly this error... https://readthedocs.com/projects/icb-scanpy/builds/272229/

Reply:
(The build shows how the docs broke after this commit.)

Reply:
I don't know, the exact same thing works in many other locations in the docs, I played around with it quite a bit. For now, in order to have the docs build, I restored the previous inline solution.

Reply:
I’ve seen an error like this in relation to autosummary, where it dumbly creates a summary from the raw first paragraph of function docs instead of first resolving indirect references.

Why does this happen here?